src/contexts/AuthContext.tsx

```diff
@@ -0,0 +1,40 @@
+import { createContext, useCallback, useState } from "react";
+
+import sessionHandler from "../utils/handlers/session.handler";
+
+
+interface AuthContextValue {
```

Comment:
Não é melhor separar AuthContextValue em uma variavel a parte para seguirmpos uma padronização? Aí seguimos um padrão parecido com o de Components, o que acha?

Reply:
Por ser a interface do contexto (que em teoria só se usa nesse arquivo), de repente deixar aqui pode ser legal tbm pra deixar tudo próximo. Por ser perfumaria, vai do que vcs decidirem como padrão.

Reply:
Sim, mas aí ficaria em um pasta específica pra esse contexto, em um arquivo específico para interface desse contexto... acho que fica mais organizado principalmente pq acho que vamos adicionar outros contextos depois

Reply:
Algo assim

![image](https://github.com/The-Bugger-Ducks/mood-hound-web/assets/78885451/53b73d3b-a7d1-48b0-9c0e-fc7c67084e36)

Reply:
Prefiro manter nesse arquivo, na mesma linha do pensamento da Maria. Gosto de colocar em um arquivo de interface quando vou utilizar em outros lugares, nesse caso acho que não se aplica.